dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -130,7 +130,9 @@
   = form_for(resource, as: resource_name, url: registration_path(resource_name), html: {method: :put, id: 'lti-sync-settings-form'}) do |f|
     = f.hidden_field :lti_roster_sync_enabled
 
-- if current_user.student? && experiment_value('cpa-partial-lockout', request) && Cpa.cpa_experience(request) && Policies::ChildAccount.user_predates_policy?(current_user)
+-# The CPA checks for whether or not the account is a personal login... so we check everything except that
+- locked = experiment_value('cpa-partial-lockout', request) && Policies::ChildAccount.user_predates_policy?(current_user, assume_permission_required: true)
```

Comment:
could change this to:

```
- locked = experiment_value('cpa-partial-lockout', request) && Policies::ChildAccount.underage?(current_user) && !Policies::ChildAccount::ComplianceState.permission_granted?(current_user)
```

which I think is the same idea: We have a partial lockout (can see the account settings) where you cannot link accounts or create a personal login.

This `locked` when true will show the parental permission field and lock the personal account creation until you are granted permission (or become old enough).